### PR TITLE
improvements

### PR DESCRIPTION
**Pull request by etokareva**

### CODE REVIEW

File: src/core/edit/edit.model.js [hunk 1]
```diff
@@ -11,4 +11,19 @@ export default class EditModel {
 		this.cancel = new Command();
 		this.reset = new Command();
 	}
+
+	get commitShortcut() {
```
Comment (klumba12) [inline src/core/edit/edit.model.js]: model should have only commitShortcuts option

Comment (klumba12) [reply src/core/edit/edit.model.js]: all other should be in controller

File: src/core/edit/edit.model.js [hunk 2]
```diff
@@ -10,5 +10,11 @@ export default class EditModel {
 		this.commit = new Command();
 		this.cancel = new Command();
 		this.reset = new Command();
+		this.commitShortcuts = {
+			'$default': 'tab|enter',
```
Comment (klumba12) [inline src/core/edit/edit.model.js]: please for the date add ctrl+s

Comment (klumba12) [reply src/core/edit/edit.model.js]: for the text tab is ok too, also tab should go to the next cell from edit mode does it work?

Comment (etokareva) [reply src/core/edit/edit.model.js]: With tabs and shift tabs everything is ok. ctrl+s causes saving of the html also as commiting of the text/number/password templates. But date template seems to not work with ctrl+s

Comment (vkorolev) [reply src/core/edit/edit.model.js]: `ctrl+s` is a common system shortcut, so we I don't think we need to intercept this

**Review by klumba12 — APPROVED** [review]

**Pull request merged**